doc/Administration-guide/Tenants-and-applications-management/index.md

```diff
@@ -10,83 +10,75 @@ sort_idx: 40
 * TOC
 {:toc}
 
+To follow this guide, you need to have your [Kaa platform installed]({{root_url}}Administration-guide/System-installation/) and the [Administration UI]({{root_url}}Glossary/#administration-ui) available from the web.
 
-This guide assumes that Kaa has already been installed and Admin UI is available from the web. If it's not the case, look at the [Installation guide]({{root_url}}Administration-guide/System-installation/) for more info.
+## Kaa users
 
-# First registration
+The first time you log in to Administration UI, it is required that you register as a Kaa administrator user.
 
-The first time you log in to Kaa Admin UI, it is required that you register as a [Kaa admin](#kaa-admin) user.
+![](attach/image2015-5-28%2017-7-0.png)
 
-<img src="attach/image2015-5-28%2017-7-0.png" width="400" height="270">
+The following three user types are available in Kaa:
 
-# Kaa user types
+- [Kaa administrator]({{root_url}}Glossary/#kaa-administrator)
+- [Tenant administrator]({{root_url}}Glossary/#tenant-administrator)
+- [Tenant developer]({{root_url}}Glossary/#tenant-developer)
 
-The following three user types are available in Kaa :
+>**NOTE:** For all the above users, [Kaa Sandbox]({{root_url}}Glossary/#kaa-sandbox) provides [default credentials]({{root_url}}Getting-started/#administration-ui).
+{:.note}
 
-- Kaa admin
-- Tenant admin
-- Tenant developer
+## Kaa administrator
 
-**NOTE**
-In Kaa Sandbox, [default credentials]({{root_url}}Getting-started/#administration-ui) are provided for all three types of Kaa users.
-
-# Kaa admin
-
-The _Kaa admin_ is the highest level administrator of Kaa. He is able to create, edit, and delete tenant admins.
-To log into the Kaa UI as a Kaa admin, use the previously created username/password for the Kaa admin.
+Kaa administrator is the highest level administrator of Kaa.
+This user can create, edit, and delete tenant administrators.
+To log in to the Administration UI as a Kaa administrator, use the username and password previously created for the Kaa administrator.
```

Comment:
add   
`(or [default credentials]({{root_url}}Getting-started/#administration-ui)) `
at the end of line
because process  of creation is not described on this page